charger/keba.go

```diff
@@ -245,6 +245,21 @@ func (c *Keba) MaxCurrent(current int64) error {
 	return nil
 }
 
+// maxCurrentMillis implements the ChargerEx interface
+func (c *Keba) maxCurrentMillis(current float64) error {
```

Comment:
maxCurrentMillis muss gross sein damit go es über Packagegrenzen hinaus sieht

Reply:
Hab das bei Wallbe rauskopiert...

Reply:
Yo, aber da wird das von einem Wrapper aufgerufen: https://github.com/andig/evcc/blob/master/charger/wallbe_decorators.go#L255. Liegt daran dass zur Compilezeit noch nicht klar ist, ob das Gerät das Feature "exposed".

Reply:
Ist es dann dort ggf. auch falsch?

Reply:
Aha! Das heisst Keba braucht sowas auch oder ist es nur weil es bei Wallbe beide Varianten gibt?

Reply:
Du brauchst das damit in dieser Zeile https://github.com/andig/evcc/blob/master/core/loadpoint.go#L445 "ok" rauskommt. Kleingeschrieben ist das immer `false`.

Reply:
Verstehe (so langsam)